pkg/metrics/metrics.go

```diff
@@ -39,6 +39,9 @@ const (
 	OperatorPortName = "http-metrics"
 	// CRPortName defines the custom resource specific metrics' port name used in the metrics Service.
 	CRPortName = "cr-metrics"
+
+	PortNum   int32 = 8383
```

Comment:
These should be `.*Number` IMO.

Reply:
Maybe `PortValue` would make even more sense? 🤔